browser/src/control/Control.UIManager.ts

```diff
@@ -904,7 +904,9 @@ class UIManager extends L.Control {
 		if (button.tablet === false && window.mode.isTablet()) {
 			return;
 		}
-		if (!this.notebookbar)
+		if (button.target === 'context-toolbar')
```

Comment:
```suggestion
		if (button.target === 'contextual-toolbar')
```
I like better this wording.